devops/Dockerfile

```diff
@@ -4,10 +4,12 @@ ARG ENVKEY
 ARG APP_DIR=/app
 ARG DSHOP_BACKEND_SOURCE="backend"
 ARG DSHOP_DAPP_SOURCE="shop"
+ARG SENTRY_DSN
```

Comment:
May want to configure this with an empty default.  Or at least test that this doesn't require the `SENTRY_DSN` build arg to be defined to build the image.